docs/pages/setup/reference/backends.mdx

```diff
@@ -329,6 +330,47 @@ This is needed to detect session recordings.
 }
 ```
 
+### ACL Example: Transferring Object Ownership
```

Comment:
```suggestion
### ACL example: transferring object ownership
```

Let's make headings sentence case per the style guide (https://goteleport.com/docs/docs/best-practices/#style-guide)